core/services/relay/evm/evm.go

```diff
@@ -129,12 +151,20 @@ func newConfigProvider(lggr logger.Logger, chainSet evm.ChainSet, args relaytype
 		ChainID:         chain.Config().ChainID().Uint64(),
 		ContractAddress: contractAddress,
 	}
+
+	ctx, cancel := context.WithCancel(context.Background())
 	return &configWatcher{
+		lggr:             lggr,
 		contractAddress:  contractAddress,
 		contractABI:      contractABI,
 		configPoller:     configPoller,
 		offchainDigester: offchainConfigDigester,
 		chain:            chain,
+		new:              args.New,
+		fromBlock:        relayConfig.FromBlock,
```

Comment:
what's the default for this if not present in config?

Reply:
zero, meaning unspecified and no replay